acceptance/testdata/mock_app/project.toml

```
@@ -0,0 +1,3 @@
+[project]
+  version = "1.0.2"
+  source-url = "https://github.com/buildpacks/pack"
```

```suggestion
  source-url = "https://github.com/buildpacks/pack"

```

I hate this but we should have a new line character at the end of each file. See https://unix.stackexchange.com/q/18743